File: src/etch/bindings/csharp/transport/MessagizerMessageSource.cs
using System;
using System.Text;
using System.Collections.Generic;
using Etch.Support;

namespace Etch.Transport
{
    /// <summary>
    /// A combination of MessageSource and Source.
    /// </summary>
    public interface MessagizerMessageSource : Source, MessageSource
    {
        // nothing
    }
}

File: src/etch/bindings/java/transport/MessagizerMessageSource.java
/**
 * 
 */
package etch.bindings.java.transport;

import metreos.core.io.Source;
import etch.bindings.java.support.MessageSource;

/**
 * A combination of MessageSource and Source.
 */
public interface MessagizerMessageSource
	extends MessageSource,
		Source<MessageHandler<MessagizerMessageSource>>
{
	// nothing
}
